hack/make-aggregator-pki.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
What would you think about adding a new folder to the `devenv` folder, with files relating to this project?

Reply:
The [readme](https://github.com/grafana/grafana/blob/main/hack/README.md) in that folder explains the current reason for its existence. I piggy-backed on top of its existence and added my script there.

On an initial look at what devenv related targets do inside the Makefile, I see they are powered by docker-compose. As such, do you think, this is adjacent enough?

Also, might be better to wait to nail this down once we have more clarity on the aggregator usage in Grafana. It could potentially converge to not being a command.

Reply:
Ah! I missed that readme, thanks for sharing that with me :)

Makes sense to keep it as-is, thanks for clarifying!